modules/component/src/componentNext.ts

```diff
@@ -170,6 +201,38 @@ export class ComponentNext<P1 extends ComponentProps> {
     )
   }
 
+  /**
+   * Adds child component to a component, this operator does multiple things i.e
+   *
+   * ```typescript
+   * import {ComponentNext} from '@action-land/component'
+   *
+   * const child1 = ComponentNext.lift({c1: 100})
+   * const child2 = ComponentNext.lift({c2: 200})
+   * const component = ComponentNext.lift({c: 1000})
+   *  .install(
+   *  {
+   *    child1,
+   *    child2
+   *  }
+   * )
+   * // Init state becomes
```

Comment:
- Move the comment and everything to before the code example. Write it as a paragraph if need be, since there are so many effects
- after that the code-sample will be easier to parse imo